### ISSUE TITLE
Invalid URI

### ISSUE THREAD

**Issue by SSebigo** [na-ji/node-manga-parser#1]
Hi, I've tried your module, and when I use any methods after `getPopularMangaList` (like this: 
```javascript
Parser.getPopularMangaList("mangafox")
  .then(paginator => {
    const manga = paginator.mangas[0];

    return Parser.getMangaDetail("mangafox", manga);
  })
  .then(manga => {
    console.log(manga);
  })
```
 )
I got this error `Error: Invalid URI "//mangafox.la/manga/name_of_the_manga/"` in the actual code it occurred at `getMangaDetail`, but as said above it occurs with other methods.

PS: when I take this URI and past it, in my browser, it seems to be a good one.

**Comment by na-ji** [comment]
Hello @SSebigo !

Thanks for the report, I will look into it

**Comment by na-ji** [comment]
I just published version `1.4.0`, which should fix the issue. If the issue is still not fixed, please re-open the issue.
